src/renderer/models/account.ts

```diff
@@ -112,6 +112,9 @@ export class CGAccount implements IAccount {
     public addValidator(validator: Keypair): void {
         this.validators.push(validator);
     }
+    public removeValidator(index: number): void {
+        this.validators.splice(index, 1);
```

Comment:
But if you login again you will have that validator again?

I think we need to delete physical keystore as well

Reply:
It deletes physical keystore in Dashboard with .destroy() method.
I made this remove validator method just to remove it from CGAccount saved in redux.

`const selectedV4Keystore = new V4Keystore(
                path.join(validatorsData.directory,selectedValidatorPublicKey + ".json"));
            selectedV4Keystore.destroy();
            props.auth.auth.removeValidator(selectedValidatorIndex);
            props.storeAuth(props.auth.auth);`